_includes/events-hero.html

```diff
@@ -0,0 +1,40 @@
+{% macro render(hero) %}
+
+{% from "post-macros.html" import category_icon as category_icon %}
+
+<section class="content_hero hero">
+    <div class="hero_wrapper">
+        <a href="#" class="hero_card hero_card__events">
```

Comment:
My perspective on the `section` question is this: The `<section class="content_hero hero">` that wraps the whole thing achieves this. Making `section.hero_card` would mean that it (`section.hero_card`) is the lone sectioning element inside of its parent (`section.content_hero.hero`). So isn't it just redundant?

Now I could see making `.hero_card` the `section` and changing the outer `.content_hero.hero` to a **`div`**, to get the `section` closer to the actual content, if that appeals to you.

Does that make sense? Please feel free to rebut. I may have a flawed understanding of sectioning elements and document outlines. It's something we've struggled to be consistent about. I love a good HTML semantics debate :grinning: (truly!)